how-to-guides/http-to-td.md

```diff
@@ -25,9 +25,7 @@ For simplicity, this article will describe how to set up a one-node configuratio
 
 The TD Output plugin is included in Fluentd's deb/rpm package \(`td-agent`\) by default. If you want to use RubyGems to install the plugin, please use `gem install fluent-plugin-td`.
```

Comment:
```suggestion
The TD Output plugin is included in Fluentd's deb/rpm package \(`fluent-package`\) by default. If you want to use RubyGems to install the plugin, please use `gem install fluent-plugin-td`.
```

TD plugin is still included in fluent-package, so we can simply replace it.

Reply:
Sorry, it's not appropriate in this PR.
We need to modify configurations too if we replace it.